This is EMC flake model

YihuaWu-NOAA opened the pull request:
Updated NCAR ccpp physics
Added flake

Associated PRs:

https://github.com/NCAR/ccpp-physics/pull/471  
https://github.com/NOAA-EMC/fv3atm/pull/141 
https://github.com/ufs-community/ufs-weather-model/pull/164

climbfuji commented on parm/ccpp_v16beta_flake_c96.nml.IN:
```diff
@@ -0,0 +1,338 @@
+﻿&amip_interp_nml
```
Could this parm file be joined with the existing GFS v16 beta parm file, using variable substitutions?

climbfuji commented on tests/fv3_conf/ccpp_gfs_v16_flake_run.IN:
```diff
@@ -0,0 +1,28 @@
+
+cp ${PATHRT}/../FV3/ccpp/suites/suite_${CCPP_SUITE}.xml suite_${CCPP_SUITE}.xml
+
+# Add path to libccpp.so and libccpphys.so to LD_LIBRARY_PATH, append to module-setup.sh
```
This block (lines 24-28) is no longer needed, please remove.

climbfuji commented on tests/fv3_conf/ccpp_gfs_v16_flake_run.IN:
```diff
@@ -0,0 +1,28 @@
+rm -fr INPUT RESTART
```
Could this regression test setup-run-dir script be joined with an existing script, using variables to control copying the ORO_FLAKE data?

junwang-noaa replied:
I'd prefer to have this new feature separate from gfs_v16 as the feature does not belong to gfs_v16, but based on gfs_v16. In the future it can be one of candidate to be included in next production branch, srw, RRFS, or gfsv17.

climbfuji replied:
Ok, all good.

climbfuji commented on tests/rt.conf:
```diff
@@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
 COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
 
 RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
+######################################################################################################################################################################################
```
Is it possible to combine this COMPILE line with a previous one (for example where the original GFSv16beta suite is compiled) to avoid an extra compile step?

climbfuji replied:
Also, please add this regression test to `rt_gnu.conf` as well. @junwang-noaa @DusanJovic-NOAA

YihuaWu-NOAA replied:
Yes, I did it

YihuaWu-NOAA commented:
Hi, Dom,

In parm/ccpp_v16beta_flake_c96.nml.IN
<https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
:

> @@ -0,0 +1,338 @@
+﻿&amip_interp_nml

Could this parm file be joined with the existing GFS v16 beta parm
file, using variable substitutions?


Yes.  Could you show me How can I do it.  I am familiar  with the
setup of these files.

Thanks! Regards





*Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
Systems Group
NOAA/NWS/NCEP/EMC
5830 University Research Court, Room: 2031
College Park, Maryland 20740, USA
Phone: (301)-683-3691
Fax:       (301)-683-3703


On Thu, Jul 16, 2020 at 9:14 AM Dom Heinzeller <notifications@github.com>
wrote:

> *@climbfuji* requested changes on this pull request.
> ------------------------------
>
> In parm/ccpp_v16beta_flake_c96.nml.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
> :
>
> > @@ -0,0 +1,338 @@
>
> +﻿&amip_interp_nml
>
>
> Could this parm file be joined with the existing GFS v16 beta parm file,
> using variable substitutions?
> ------------------------------
>
> In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774677>
> :
>
> > +fi
>
> +cp    @[RTPWD]/FV3_input_data/INPUT/aerosol.dat .
>
> +cp    @[RTPWD]/FV3_input_data/INPUT/co2historicaldata_201*.txt .
>
> +cp    @[RTPWD]/FV3_input_data/INPUT/sfc_emissivity_idx.txt .
>
> +cp    @[RTPWD]/FV3_input_data/INPUT/solarconstant_noaa_an.txt .
>
> +cp    @[RTPWD]/FV3_input_data/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
>
> +cp    @[RTPWD]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
>
> +cp    @[RTPWD]/FV3_input_data/*grb .
>
> +cp    @[RTPWD]/FV3_input_data/*_table .
>
> +cp    @[RTPWD]/FV3_input_data/diag_table_gfdlmp diag_table
>
> +cp    @[RTPWD]/FV3_input_data_gsd/field_table_suite2 field_table
>
> +cp    @[RTPWD]/FV3_input_data/*configure .
>
> +
>
> +cp ${PATHRT}/../FV3/ccpp/suites/suite_${CCPP_SUITE}.xml suite_${CCPP_SUITE}.xml
>
> +
>
> +# Add path to libccpp.so and libccpphys.so to LD_LIBRARY_PATH, append to module-setup.sh
>
>
> This block (lines 24-28) is no longer needed, please remove.
> ------------------------------
>
> In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775016>
> :
>
> > @@ -0,0 +1,28 @@
>
> +rm -fr INPUT RESTART
>
>
> Could this regression test setup-run-dir script be joined with an existing
> script, using variables to control copying the ORO_FLAKE data?
> ------------------------------
>
> In tests/rt.conf
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775415>
> :
>
> > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
>
>  COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
>
>
>
>  RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
>
> +######################################################################################################################################################################################
>
>
> Is it possible to combine this COMPILE line with a previous one (for
> example where the original GFSv16beta suite is compiled) to avoid an extra
> compile step?
> ------------------------------
>
> In tests/rt.conf
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775733>
> :
>
> > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
>
>  COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
>
>
>
>  RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
>
> +######################################################################################################################################################################################
>
>
> Also, please add this regression test to rt_gnu.conf as well.
> @junwang-noaa <https://github.com/junwang-noaa> @DusanJovic-NOAA
> <https://github.com/DusanJovic-NOAA>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/164#pullrequestreview-449832125>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANJBYQHL2EJ7N4OGDBNVYE3R334MTANCNFSM4OW5GIQA>
> .
>

climbfuji commented:
Look at parm/ccpp_gsd.nml.IN for example. You see many examples of variables like @[LHEATSTRG] that get substituted with actual values.

The default values are set in tests/default_vars.sh - regression tests that need to change some of these values do so in their tests/tests/REGTESTNAME script.

You can diff the basic GFSv16beta parm file with yours to see what needs to be added/changed. Then add those as variables to the default GFSv16 parm file, make sure there are default values defined in tests/default_vars.sh and that the default values are correct, then overwrite them in tests/tests/YOURFLAKEREGTESTNAME.

Does this make sense?

Dom

> On Jul 16, 2020, at 8:39 AM, YihuaWu-NOAA <notifications@github.com> wrote:
> 
> 
> Hi, Dom,
> 
> In parm/ccpp_v16beta_flake_c96.nml.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
> :
> 
> > @@ -0,0 +1,338 @@
> +﻿&amip_interp_nml
> 
> Could this parm file be joined with the existing GFS v16 beta parm
> file, using variable substitutions?
> 
> 
> Yes. Could you show me How can I do it. I am familiar with the
> setup of these files.
> 
> Thanks! Regards
> 
> 
> 
> 
> 
> *Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
> Systems Group
> NOAA/NWS/NCEP/EMC
> 5830 University Research Court, Room: 2031
> College Park, Maryland 20740, USA
> Phone: (301)-683-3691
> Fax: (301)-683-3703
> 
> 
> On Thu, Jul 16, 2020 at 9:14 AM Dom Heinzeller <notifications@github.com>
> wrote:
> 
> > *@climbfuji* requested changes on this pull request.
> > ------------------------------
> >
> > In parm/ccpp_v16beta_flake_c96.nml.IN
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
> > :
> >
> > > @@ -0,0 +1,338 @@
> >
> > +﻿&amip_interp_nml
> >
> >
> > Could this parm file be joined with the existing GFS v16 beta parm file,
> > using variable substitutions?
> > ------------------------------
> >
> > In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774677>
> > :
> >
> > > +fi
> >
> > +cp @[RTPWD]/FV3_input_data/INPUT/aerosol.dat .
> >
> > +cp @[RTPWD]/FV3_input_data/INPUT/co2historicaldata_201*.txt .
> >
> > +cp @[RTPWD]/FV3_input_data/INPUT/sfc_emissivity_idx.txt .
> >
> > +cp @[RTPWD]/FV3_input_data/INPUT/solarconstant_noaa_an.txt .
> >
> > +cp @[RTPWD]/FV3_input_data/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
> >
> > +cp @[RTPWD]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
> >
> > +cp @[RTPWD]/FV3_input_data/*grb .
> >
> > +cp @[RTPWD]/FV3_input_data/*_table .
> >
> > +cp @[RTPWD]/FV3_input_data/diag_table_gfdlmp diag_table
> >
> > +cp @[RTPWD]/FV3_input_data_gsd/field_table_suite2 field_table
> >
> > +cp @[RTPWD]/FV3_input_data/*configure .
> >
> > +
> >
> > +cp ${PATHRT}/../FV3/ccpp/suites/suite_${CCPP_SUITE}.xml suite_${CCPP_SUITE}.xml
> >
> > +
> >
> > +# Add path to libccpp.so and libccpphys.so to LD_LIBRARY_PATH, append to module-setup.sh
> >
> >
> > This block (lines 24-28) is no longer needed, please remove.
> > ------------------------------
> >
> > In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775016>
> > :
> >
> > > @@ -0,0 +1,28 @@
> >
> > +rm -fr INPUT RESTART
> >
> >
> > Could this regression test setup-run-dir script be joined with an existing
> > script, using variables to control copying the ORO_FLAKE data?
> > ------------------------------
> >
> > In tests/rt.conf
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775415>
> > :
> >
> > > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
> >
> > COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp | standard | jet.intel | fv3 |
> >
> >
> >
> > RUN | fv3_ccpp_gocart_clm | standard | | fv3 |
> >
> > +######################################################################################################################################################################################
> >
> >
> > Is it possible to combine this COMPILE line with a previous one (for
> > example where the original GFSv16beta suite is compiled) to avoid an extra
> > compile step?
> > ------------------------------
> >
> > In tests/rt.conf
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775733>
> > :
> >
> > > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
> >
> > COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp | standard | jet.intel | fv3 |
> >
> >
> >
> > RUN | fv3_ccpp_gocart_clm | standard | | fv3 |
> >
> > +######################################################################################################################################################################################
> >
> >
> > Also, please add this regression test to rt_gnu.conf as well.
> > @junwang-noaa <https://github.com/junwang-noaa> @DusanJovic-NOAA
> > <https://github.com/DusanJovic-NOAA>
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/ufs-community/ufs-weather-model/pull/164#pullrequestreview-449832125>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/ANJBYQHL2EJ7N4OGDBNVYE3R334MTANCNFSM4OW5GIQA>
> > .
> >
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub <https://github.com/ufs-community/ufs-weather-model/pull/164#issuecomment-659456447>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB5C2RKHHNFAHT4V4HEWKULR34GLZANCNFSM4OW5GIQA>.
>

YihuaWu-NOAA commented:
I removed line 24-28 in  tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
<https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774677>
I combined the compiling lines for flake
I added the  regression test to rt_gnu.conf .
Thanks!




*Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
Systems Group
NOAA/NWS/NCEP/EMC
5830 University Research Court, Room: 2031
College Park, Maryland 20740, USA
Phone: (301)-683-3691
Fax:       (301)-683-3703


On Thu, Jul 16, 2020 at 10:39 AM Yihua. Wu - NOAA Affiliate <
yihua.wu@noaa.gov> wrote:

> Hi, Dom,
>
> In parm/ccpp_v16beta_flake_c96.nml.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
> :
>
> > @@ -0,0 +1,338 @@
> +﻿&amip_interp_nml
>
> Could this parm file be joined with the existing GFS v16 beta parm file, using variable substitutions?
>
>
> Yes.  Could you show me How can I do it.  I am familiar  with the setup of these files.
>
> Thanks! Regards
>
>
>
>
>
> *Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
> Systems Group
> NOAA/NWS/NCEP/EMC
> 5830 University Research Court, Room: 2031
> College Park, Maryland 20740, USA
> Phone: (301)-683-3691
> Fax:       (301)-683-3703
>
>
> On Thu, Jul 16, 2020 at 9:14 AM Dom Heinzeller <notifications@github.com>
> wrote:
>
>> *@climbfuji* requested changes on this pull request.
>> ------------------------------
>>
>> In parm/ccpp_v16beta_flake_c96.nml.IN
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774335>
>> :
>>
>> > @@ -0,0 +1,338 @@
>>
>> +﻿&amip_interp_nml
>>
>>
>> Could this parm file be joined with the existing GFS v16 beta parm file,
>> using variable substitutions?
>> ------------------------------
>>
>> In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455774677>
>> :
>>
>> > +fi
>>
>> +cp    @[RTPWD]/FV3_input_data/INPUT/aerosol.dat .
>>
>> +cp    @[RTPWD]/FV3_input_data/INPUT/co2historicaldata_201*.txt .
>>
>> +cp    @[RTPWD]/FV3_input_data/INPUT/sfc_emissivity_idx.txt .
>>
>> +cp    @[RTPWD]/FV3_input_data/INPUT/solarconstant_noaa_an.txt .
>>
>> +cp    @[RTPWD]/FV3_input_data/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
>>
>> +cp    @[RTPWD]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
>>
>> +cp    @[RTPWD]/FV3_input_data/*grb .
>>
>> +cp    @[RTPWD]/FV3_input_data/*_table .
>>
>> +cp    @[RTPWD]/FV3_input_data/diag_table_gfdlmp diag_table
>>
>> +cp    @[RTPWD]/FV3_input_data_gsd/field_table_suite2 field_table
>>
>> +cp    @[RTPWD]/FV3_input_data/*configure .
>>
>> +
>>
>> +cp ${PATHRT}/../FV3/ccpp/suites/suite_${CCPP_SUITE}.xml suite_${CCPP_SUITE}.xml
>>
>> +
>>
>> +# Add path to libccpp.so and libccpphys.so to LD_LIBRARY_PATH, append to module-setup.sh
>>
>>
>> This block (lines 24-28) is no longer needed, please remove.
>> ------------------------------
>>
>> In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775016>
>> :
>>
>> > @@ -0,0 +1,28 @@
>>
>> +rm -fr INPUT RESTART
>>
>>
>> Could this regression test setup-run-dir script be joined with an
>> existing script, using variables to control copying the ORO_FLAKE data?
>> ------------------------------
>>
>> In tests/rt.conf
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775415>
>> :
>>
>> > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
>>
>>  COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
>>
>>
>>
>>  RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
>>
>> +######################################################################################################################################################################################
>>
>>
>> Is it possible to combine this COMPILE line with a previous one (for
>> example where the original GFSv16beta suite is compiled) to avoid an extra
>> compile step?
>> ------------------------------
>>
>> In tests/rt.conf
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r455775733>
>> :
>>
>> > @@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
>>
>>  COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
>>
>>
>>
>>  RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
>>
>> +######################################################################################################################################################################################
>>
>>
>> Also, please add this regression test to rt_gnu.conf as well.
>> @junwang-noaa <https://github.com/junwang-noaa> @DusanJovic-NOAA
>> <https://github.com/DusanJovic-NOAA>
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ufs-community/ufs-weather-model/pull/164#pullrequestreview-449832125>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ANJBYQHL2EJ7N4OGDBNVYE3R334MTANCNFSM4OW5GIQA>
>> .
>>
>

YihuaWu-NOAA commented:
Yes, sure.
Change it to ccpp_gfs_flake?




*Yihua----------------------------------------------------Yihua Wu, Ph D*I.M.
Systems Group
NOAA/NWS/NCEP/EMC
5830 University Research Court, Room: 2031
College Park, Maryland 20740, USA
Phone: (301)-683-3691
Fax:       (301)-683-3703


On Fri, Jul 17, 2020 at 9:53 AM Jun Wang <notifications@github.com> wrote:

> *@junwang-noaa* commented on this pull request.
> ------------------------------
>
> In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r456456848>
> :
>
> > @@ -0,0 +1,28 @@
> +rm -fr INPUT RESTART
>
> I'd prefer to have this new feature separate from gfs_v16 as the feature
> does not belong to gfs_v16, but based on gfs_v16. In the future it can be
> one of candidate to be included in next production branch, srw, RRFS, or
> gfsv17.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r456456848>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANJBYQAHCU6I3QZOXVEILLLR4BJW5ANCNFSM4OW5GIQA>
> .
>

junwang-noaa commented:
I'd suggest leaving it as is since we know it is a new feature added based
on gfs_v16.

On Fri, Jul 17, 2020 at 10:02 AM YihuaWu-NOAA <notifications@github.com>
wrote:

> Yes, sure.
> Change it to ccpp_gfs_flake?
>
>
>
>
> *Yihua----------------------------------------------------Yihua Wu, Ph
> D*I.M.
> Systems Group
> NOAA/NWS/NCEP/EMC
> 5830 University Research Court, Room: 2031
> College Park, Maryland 20740, USA
> Phone: (301)-683-3691
> Fax: (301)-683-3703
>
>
> On Fri, Jul 17, 2020 at 9:53 AM Jun Wang <notifications@github.com> wrote:
>
> > *@junwang-noaa* commented on this pull request.
> > ------------------------------
> >
> > In tests/fv3_conf/ccpp_gfs_v16_flake_run.IN
> > <
> https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r456456848
> >
> > :
> >
> > > @@ -0,0 +1,28 @@
> > +rm -fr INPUT RESTART
> >
> > I'd prefer to have this new feature separate from gfs_v16 as the feature
> > does not belong to gfs_v16, but based on gfs_v16. In the future it can be
> > one of candidate to be included in next production branch, srw, RRFS, or
> > gfsv17.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ufs-community/ufs-weather-model/pull/164#discussion_r456456848
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ANJBYQAHCU6I3QZOXVEILLLR4BJW5ANCNFSM4OW5GIQA
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/164#issuecomment-660123892>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI7D6TKFMAU3II4Z2D2T4HDR4BKWRANCNFSM4OW5GIQA>
> .
>
